index.js

```diff
@@ -60,12 +62,13 @@ class ShareSheet extends React.Component<Props> {
     return false;
   }
   render() {
+    const { style = {}, overlayStyle = {}, ...props } = this.props;
```

Comment:
Why did you used that `...props`  on props?

Reply:
Because style and overlayStyle was not meant to be passed to Overlay